static/directives/repo-tag-history.html

```diff
@@ -20,7 +21,7 @@
         </thead>
         <tbody>
           <tr style="height: 20px;"><td colspan="4"></td></tr>
-          <tr ng-repeat="entry in historyEntries" class="history-entry" ng-class="getEntryClasses(entry, filter)">
+          <tr ng-repeat="entry in historyEntries" class="history-entry" ng-class="getEntryClasses(entry, filter, options.showFuture)">
```

Comment:
The 3rd argument isn't being used in the function itself (only accepts 2 parameters).

Reply:
Whoops... forgot to remove that. Done.